tests/compile-fail/stacked_borrows/interior_mut1.rs

```diff
@@ -3,8 +3,13 @@ use std::cell::UnsafeCell;
 fn main() { unsafe {
     let c = &UnsafeCell::new(UnsafeCell::new(0));
     let inner_uniq = &mut *c.get();
-    let inner_shr = &*inner_uniq; // a SharedRW with a tag
-    *c.get() = UnsafeCell::new(1); // invalidates the SharedRW
+    // stack: [c: SharedReadWrite, inner_uniq: Unique]
```

Comment:
Would you like a scheme that allows you to write such comments (or attributes or sth) and have miri assert them against the actual stacks?

Reply:
That would be pretty cool, but also sounds extremely hard. In particular, the real stack will have more items between the ones I am writing because of all the implicit reborrowing that is going on.

Reply:
a simpler way would be to use ui tests to just dump the diff of all stacks at every step

Reply:
Then the useful information will drown in the noise.